datalake/src/main/java/com/sequenceiq/datalake/flow/dr/DatalakeDatabaseDrEvent.java

```diff
@@ -0,0 +1,32 @@
+package com.sequenceiq.datalake.flow.dr;
+
+import com.sequenceiq.flow.core.FlowEvent;
+
+public enum DatalakeDatabaseDrEvent implements FlowEvent {
+
+    DATALAKE_DATABASE_BACKUP_EVENT("DATALAKE_DATABASE_BACKUP_EVENT"),
+    DATALAKE_DATABASE_BACKUP_COULD_NOT_START_EVENT("DATALAKE_DATABASE_BACKUP_COULD_NOT_START_EVENT"),
+    DATALAKE_DATABASE_BACKUP_IN_PROGRESS_EVENT("DATALAKE_DATABASE_BACKUP_IN_PROGRESS_EVENT"),
+    DATALAKE_DATABASE_BACKUP_SUCCESS_EVENT("DATALAKE_DATABASE_BACKUP_SUCCESS_EVENT"),
+    DATALAKE_DATABASE_BACKUP_FAILED_EVENT("DATALAKE_DATABASE_BACKUP_FAILED_EVENT"),
+    DATALAKE_DATABASE_BACKUP_FINALIZED_EVENT("DATALAKE_DATABASE_BACKUP_FINALIZED_EVENT"),
+    DATALAKE_DATABASE_BACKUP_FAILURE_HANDLED_EVENT("DATALAKE_DATABASE_BACKUP_FAILURE_HANDLED_EVENT"),
+    DATALAKE_DATABASE_RESTORE_EVENT("DATALAKE_DATABASE_RESTORE_EVENT"),
+    DATALAKE_DATABASE_RESTORE_COULD_NOT_START_EVENT("DATALAKE_DATABASE_RESTORE_COULD_NOT_START_EVENT"),
+    DATALAKE_DATABASE_RESTORE_IN_PROGRESS_EVENT("DATALAKE_DATABASE_RESTORE_IN_PROGRESS_EVENT"),
+    DATALAKE_DATABASE_RESTORE_SUCCESS_EVENT("DATALAKE_DATABASE_RESTORE_SUCCESS_EVENT"),
+    DATALAKE_DATABASE_RESTORE_FAILED_EVENT("DATALAKE_DATABASE_RESTORE_FAILED_EVENT"),
+    DATALAKE_DATABASE_RESTORE_FINALIZED_EVENT("DATALAKE_DATABASE_RESTORE_FINALIZED_EVENT"),
+    DATALAKE_DATABASE_RESTORE_FAILURE_HANDLED_EVENT("DATALAKE_DATABASE_RESTORE_FAILURE_HANDLED_EVENT");
```

Comment:
please check my comment on one of your event, and use there and here also `EventSelectorUtil` instead of duplicating string everywhere

Reply:
I do not have classes defined for all the events. For the events for which classes are defined, i will make changes following your suggestion.

Reply:
you don't have to create classes for all events. I didn't want to go through all of them to tag which should be replaced. Leaving those untouched which don't have class is fine